test/unit/player.test.js

```diff
@@ -247,15 +247,15 @@ test('should hide the poster when play is called', function() {
   });
 
   equal(player.hasStarted(), false, 'the show poster flag is true before play');
-  player.play();
+  player.tech_.trigger('play');
```

Comment:
I was originally getting an error about the method currentSrc() not existing(likely since the tests use a fake tech.) I looked at how some of the other tests were written and saw that they almost exclusively trigger play in this way so I went with this instead.